### PR TITLE
add document for packages

### PR DESCRIPTION
**Pull request by cep-ter**

### CODE REVIEW

File: bluto/doc.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*Package bluto is connection pool wraper which can borrow redis connection from pool and return them when finished.
```
Comment (bbmokhtari) [inline bluto/doc.go:1]: > Package Bluto is a Redis connection pool which can lend Redis connections. The connection and the pool wrap and enhance their Redigo counterparts.

I think this is a better description as it doesn't involve in returning the connection and only in lending them.  (borrow is the reverse)
Also wraper => wrapper

File: bluto/doc.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+Pooler manage and reuse existing conection and provide advanced config options for handling timeouts.
```
Comment (bbmokhtari) [inline bluto/doc.go:2]: Pooler is no more. Use Bluto.

Typo:
manage => manages
reuse => reuses
connection => connections
for handling timeouts. => e.g. for handling timeouts.

File: commander/doc.go [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*Package commander is bassicly a connection to redis which support for multipe chained commands,
```
Comment (bbmokhtari) [inline commander/doc.go:1]: > Package Commander provides a Redis connection which can run multiple Redis commands by chaining. The connection wraps and enhances its Redigo counterpart.

Typo:
bassicly => basically
support for multipe chained commands => supports multipe chained commands

File: commander/doc.go [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+and returning the connection to pool when finished.
```
Comment (bbmokhtari) [inline commander/doc.go:2]: and it can return the connection to the pool when finished.

File: doc.go [hunk 5]
```diff
@@ -0,0 +1,17 @@
+/*package bluto is redis library wraper over redigo which add type safty, chained command and
```
Comment (bbmokhtari) [inline doc.go:1]: wraper => wrapper
which add => which adds
safty => safety
chained command => chained commands

File: doc.go [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+managed connection pool which makes it easier for developer to use the library and reduce runtime bugs.
```
Comment (bbmokhtari) [inline doc.go:2]: developer => developers

File: doc.go [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+bluto instance is compilitly thread-safe which means you can pass bluto object
```
Comment (bbmokhtari) [inline doc.go:4]: compilitly => completely
pass bluto object => pass bluto objects/pass a bluto object

File: doc.go [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+to diffretent go-routines without race condition it also automaticly return finished
```
Comment (bbmokhtari) [inline doc.go:5]: go-routines => goroutines
without race condition it => without race condition, it
automaticly => automatically
return => returns

File: doc.go [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+connection to connection pool.
```
Comment (bbmokhtari) [inline doc.go:6]: connection to => connections to

File: doc.go [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+pool also support config for connections like timeout and health-check for better managment.
```
Comment (bbmokhtari) [inline doc.go:8]: support => supports
managment => management

File: doc.go [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+commander is borrowed from bluto instance which can be used to chain multipe commands toghether and
```
Comment (bbmokhtari) [inline doc.go:10]: multipe => multiple
toghether => together

File: doc.go [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+run all of them with commit.This chaining reduce the time when the connection is kept so it improves
```
Comment (bbmokhtari) [inline doc.go:11]: .This => . This
reduce => reduces

**Review by bbmokhtari — APPROVED** [review]

**Review by rezakamalifard — APPROVED** [review]

**Pull request merged**